centreon/www/front_src/src/Header/sharedUI/ResourceCounters.tsx

```diff
@@ -0,0 +1,74 @@
+import StatusCounter from "./StatusCounter";
+import { Link } from "react-router-dom";
+
+import { makeStyles } from "tss-react/mui";
+import { SeverityCode } from "@centreon/ui";
+
+const useStyles = makeStyles()((theme) => ({
+  container: {
+    listStyle: "none",
+    padding: 0,
+    margin: 0,
+    display: "inline-block",
+    lineHeight: 1,
+    [theme.breakpoints.down(768)]: {
+      flexFlow: "row wrap",
+    },
+  },
+  item: {
+    display: "inline-block",
+    paddingRight: theme.spacing(0.25),
+  },
+  splitter: {
+    display: "none",
+    [theme.breakpoints.down(768)]: {
+      display: "block",
+      marginBottom: "2px",
+    },
+  },
+  link: {
+    textDecoration: "none",
+  },
+}));
+
+export interface CounterProps {
+  counters: {
+    onClick: (e: React.MouseEvent) => void;
+    severityCode: SeverityCode;
+    count: string | number;
+    ariaLabel: string;
+    to: string;
+  }[];
+}
+
+export default ({ counters }: CounterProps): JSX.Element => {
+  const { classes } = useStyles();
+
+  return (
+    <ul className={classes.container}>
+      {counters.map(
+        ({ to, ariaLabel, onClick, count, severityCode }, index) => (
+          <>
+            {index === 2 && (
```

Comment:
Why the third element must have a different style?

Reply:
it allow putting further items on the line.